Source/utils/display.cpp

```diff
@@ -127,7 +127,7 @@ bool SpawnWindow(const char *lpWindowName)
 	scePowerSetArmClockFrequency(444);
 #endif
 
-#if SDL_VERSION_ATLEAST(2, 0, 6)
+#if SDL_VERSION_ATLEAST(2, 0, 6) && !defined(__ANDROID__)

```

Comment:
I wonder why we disable these in the first place

Reply:
Vita using the touch as buttons

Reply:
Perhaps `&& defined(__vita__)` is better than `&& !defined(__ANDROID)`?